subprojects/packagefiles/liburing/test/meson.build

```diff
@@ -127,28 +151,30 @@ if has_array_bounds
     xcflags = xcflags + ['-Warray-bounds=0']
 endif
 
-foreach t : all_tests
-    test_name = t[0]
-    test_source = t[0] + '.' + t[1]
+test_dependencies = [thread_dep]
+fs = import('fs')
+
+foreach test_source : all_tests
+    test_name = fs.stem(test_source)
     executable(test_name,
                [test_source, 'helpers.c'],
                c_args: xcflags,
                cpp_args: xcflags,
                include_directories : inc,
                link_with : liburing.get_static_lib(),
-               dependencies : t[2],
+               dependencies : test_dependencies,
                install : true,
                install_dir : get_option('datadir') / 'liburing-test')
 
-    test(t[0],
+    test(test_name,
          runtests_sh,
-         args : t[0],
+         args : test_name,
```

Comment:
The test harness will try to detect issues logged in dmesg, but relies on not having intermixed logs in that case. So I think this should probably have used `is_parallel: false` all along?

Reply:
Since there are multiple test suites which I guess shouldn't all be run at the same time by a default `meson test`, perhaps we should use `add_test_setup()` to define a test profile which only runs the "once" tests, and declare it as the default?